primitives/xcm-bridge-hub-router/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "bp-xcm-bridge-hub-router"
```

Comment:
@svyatonik 
just nit, what about renaming just to the `bp-xcm-router` + directory to `xcm-router`,
there is only `LocalXcmChannel` which seems pretty generic,
I dont know if more code is coming here, but I dont see here anything specific to `bridge-hub`

Reply:
Don't know, my thinking was: we have our bridge pallets and pallet primitives (public types and traits, used in pallet configuration) are in the dedicated primitives crate/folder. Hence this name.

I could rename it to something that has "bridge" inside :) WDYT about `LocalBridgeHubXcmChannel`?

Reply:
ok, ok, let me walk through the whole PR first,
and then I will come back here :)

Reply:
ok, from what I see now:

I think that everywhere we use word BridgeHub, we are referencing "system parachain BridgeHub".

`modules/xcm-bridge-hub-router`:
- I thought that that `xcm-bridge-hub-router` referenced to the very specific "system parachain BridgeHub"
- but I dont see any speciality about `BridgeHub` word or something which referenced "system parachain BridgeHub", bridge is just MultiLocation in this context which we call with `SendXcm`
- so I think, maybe name `modules/xcm-bridge-router` or `modules/bridge-xcm-router` (Router to XcmBridge or XcmRouter to Bridge?) is maybe enough

`primitives/xcm-bridge-hub-router`
- my thinking was about `primitives/xcm-router` or `primitives/xcm-bridge-router` or `primitives/bridge-xcm-router` as very generic primitives (that anybody can use for their own implementation of "xcm bridge router"), not related to any specific pallet

e.g. if we check `bin/runtime-common/src/messages_xcm_extension.rs` - which looks like generic/common xcm stuff but we have [here import](https://github.com/paritytech/parity-bridges-common/pull/2294/files#diff-f94bdda84771d04e176e0b8a789e3416426ad6b76d798f04b375a7f8600e660aR30) `use bp_xcm_bridge_hub_router::LocalXcmChannel;`  which looks pretty specific to "system parachain BridgeHub"

Reply:
Not exactly. When bridge hub is mentioned, I'm mostly talking about the chain with [`pallet-xcm-bridge-hub`](https://github.com/paritytech/parity-bridges-common/tree/master/modules/xcm-bridge-hub) deployed - it supports multiple lanes, so it is technically a hub. And initially those two pallets were closely tied together, because they were need to talk to each other to ask and report bridge queues/lanes status. And maybe they'll be tied together in the future, because of https://github.com/paritytech/parity-bridges-common/pull/2313#discussion_r1281424022. However, I'm not sure yet - it needs more thinking.

There are some time anomalies with dynamic fees and dynamic lanes - the `pallet-xcm-bridge-hub` was born first, but `pallet-xcm-bridge-hub-router` will (likely) be deployed first. That said, I don't have anything against renaming those pallets to `pallet-xcm-bridge` and `pallet-xcm-bridge-router`, as you suggested. Although imo `hub` suffix gives more context here. Please comment to enact this change :)

Reply:
ok, dont need to change anything, 
"bridge hub" as "multi lane" is clear now,
but just please update Docs with this distinction to avoid confusion with "system parachain bridge hub"